pmd-rust/src/main/java/net/sourceforge/pmd/lang/rust/RustLanguageModule.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.rust;
+
+import net.sourceforge.pmd.cpd.CpdLexer;
+import net.sourceforge.pmd.lang.LanguagePropertyBundle;
+import net.sourceforge.pmd.lang.LanguageRegistry;
+import net.sourceforge.pmd.lang.rust.cpd.RustCpdLexer;
+import net.sourceforge.pmd.lang.impl.CpdOnlyLanguageModuleBase;
+
+public class RustLanguageModule extends CpdOnlyLanguageModuleBase {
+    private static final String ID = "rust";
+
+    public RustLanguageModule() {
+        super(LanguageMetadata.withId(ID).name("Rust").extensions("rust"));
```

Comment:
```suggestion
        super(LanguageMetadata.withId(ID).name("Rust").extensions("rs"));
```

Reply:
thank you! addressed in da524115c0b943256a1466c4d0145b0fc49b1a8f